bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpRendererHandler.java

```diff
@@ -590,7 +590,7 @@ public void onValueReceived(@Nullable String variable, @Nullable String value, @
                     playing = true;
                     updateState(CONTROL, PlayPauseType.PLAY);
                     scheduleTrackPositionRefresh();
-                } else if ("PAUSED_PLAYBACK".contentEquals(value)) {
+                } else if (value != null && "PAUSED_PLAYBACK".contentEquals(value)) {
```

Comment:
`contentEquals` is equivalent to `equals` when comparing two strings.
```suggestion
                } else if ("PAUSED_PLAYBACK".equals(value)) {
```